packages/medusa/src/services/totals.ts

```diff
@@ -208,14 +208,7 @@ class TotalsService extends TransactionBaseService {
     }
 
     if (opts.include_tax) {
-      if (isOrder(cartOrOrder) && cartOrOrder.tax_rate !== null) {
-        totals.original_tax_total = Math.round(
-          totals.original_tax_total * (cartOrOrder.tax_rate / 100)
-        )
-        totals.tax_total = Math.round(
-          totals.original_tax_total * (cartOrOrder.tax_rate / 100)
-        )
-      } else {
+      if (isCart(cartOrOrder) || cartOrOrder.tax_rate === null) {
```

Comment:
question: previously if it was an order with a tax rate, we were calculating the taxes with the provided tax rate to keep the old system working.

Now we have lost that feature, wdyt?

Reply:
fixed